products/compute/api.yaml

```diff
@@ -7037,6 +7059,48 @@ objects:
         resource: 'Instance'
         imports: 'selfLink'
         required: true
+      - !ruby/object:Api::Type::Array
+        name: 'storageLocations'
+        description: |
+          The RFC 4648 base64 encoded SHA-256 hash of the
```

Comment:
This description does not match `storageLocations`.

Reply:
I'll fix it.